examples/acl/src/backend/components/components.clj

```diff
@@ -0,0 +1,37 @@
+(ns components
+  (:require
+    [app]
+    [com.stuartsierra.component :as component]
+    [framework.config.core :as config]
+    [framework.db.storage :as db.storage]
+    [nrepl.server :refer [start-server stop-server]]
+    [router]
+    [web-server]))
+
+(defn system
+  [config]
+  (let [pg-cfg (:framework.db.storage/postgresql config)
+        app-cfg (:framework.app/ring config)
+        web-server-cfg (:framework.app/web-server config)
+        acl-permissions (:acl/permissions config)
+        acl-roles (:acl/roles config)]
+    (->
+      (component/system-map
+        :config config
+        :db (db.storage/postgresql pg-cfg)
+        :router (router/make-router)
+        :app (app/make-app app-cfg)
+        :acl-permissions acl-permissions
```

Comment:
roles and permissions does not have semantics of a component (start/stop)
Bind them to App component with `make-app`